modules/KalturaSupport/components/dualScreen/dualScreen.js

```diff
@@ -357,22 +366,28 @@
 					}
 				};
 
-				//Set initial view state according to configuration and playback engine
-				if ( this.getConfig( "mainViewDisplay" ) === 2 && !mw.isNativeApp() ||
+				if ( this.getConfig( "defaultDualScreenViewId" ) !== 'video-only' && !mw.isNativeApp() ||
 					this.getPlayer().isAudio()) {
 					this.bind( 'postDualScreenTransition.spinnerPostFix', function () {
 						_this.unbind( 'postDualScreenTransition.spinnerPostFix' );
 						showLoadingSlide();
 					} );
-					setTimeout( function () {
-						_this.fsm.consumeEvent( "switchView" );
-						if (_this.getPlayer().isAudio()){
-							_this.fsm.consumeEvent( "hide" );
-						}
-					}, 1000 );
 				} else {
 					showLoadingSlide();
 				}
+
+				var defaultDualScreenViewId = this.getConfig('defaultDualScreenViewId');
```

Comment:
This logic replaces the previous handling of default view id. According to Renan we should not support scenario of audio only. 

We get the default view id and updates the view accordingly